### ISSUE TITLE
second nozzle does not heat past 200 on touch screen

### ISSUE THREAD

**Issue by manganD** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2310]
.
-->
### Description
secon hotend does not heat past 200 on the touchscreen  only  of tft 70 .  second hotend is only viewable when active in a print  or when changing filament . unable to select or preheat a second hotend on touch screen. even if set in slicer second hotend is capped at 200 .  marlin screen on tft 70 everything works as it should 

### Steps to reproduce

1. when second nozzle is active for a print and you are using the touch screen  the temp is limited to 200
2.  tap nozzle icon change temp to example 220  go back to main screen temp is back to 200
3. set slicer 220 temp . put it on sd card  start print temp on second hotend limit 200
4.try to preheat second hotend   cant select second hotend 
**Expected behavior**
change temp to higher then 200 setting . the setting stays and hot end reaches 200+

**Actual behavior**
no matter how its changed the setting stays limited to 200 degrees max 

### Hardware Variant
tft 70 current firmware from this site 
### TFT Firmware Version & Main Board Firmware details
BIGTREE_TFT70_V3.0.27.x.bin . marlin  bersion 02000903

### Additional Information
[Configuration.zip](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/files/7810260/Configuration.zip)

**Comment by manganD** [comment]
no its at 275 , should i have ?

**Comment by manganD** [comment]
i am so sorry i found the solution . i went back and looked at my config.ini file  to double check the temp setting i saw that while i changed the number of extruders i missed the hotend line . i changed the hotend to 2 and now everything works as it should . i do apologize

**Comment by gordo3di** [comment]
> i am so sorry i found the solution . i went back and looked at my config.ini file to double check the temp setting i saw that while i changed the number of extruders i missed the hotend line . i changed the hotend to 2 and now everything works as it should . i do apologize

Been there

**Comment by manganD** [comment]
No I didn’t change the max temp its at 275. Should I have ?i checked when I saw you comment and  I did however see that I missed the hotend line  and didn’t change that to 2 . I changed the extruders to 2 but had missed the hotend line.  Ive since changed it and not the temps are no longer limited to 200 and I can select each hotend on the touchscreen just like I could with the marlin screen. So everything is working as it should  Sent from Mail for Windows From: gordo3diSent: Wednesday, January 5, 2022 9:25 AMTo: bigtreetech/BIGTREETECH-TouchScreenFirmwareCc: manganD; AuthorSubject: Re: [bigtreetech/BIGTREETECH-TouchScreenFirmware] second nozzle does not heat past 200 on touch screen (Issue #2310) Did you change the max temp in config.ini as well?—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***>
